contracts/modules/PoolController.sol

```diff
@@ -276,7 +276,7 @@ contract PoolController is
         pool.updatedAt = block.timestamp;
     }
 
-    function isArchivingAllowed(uint256 riskpoolId) external view returns (bool) {
+    function isArchivingAllowed(uint256 riskpoolId) external view {
```

Comment:
feels strange that an isXYZ function does not return bool. 
seems this is only used in 
- ComponentOwnerService.archive(uint256 id)
- InstanceOperatorService.archive(uint256 id)

in both cases a sequence of 3 requires would feel more natural as the nested if statement.
please consider something like

function archive(...) 
...
{
  require(comonent_type==riskpool, "ERROR...NOT_RISKPOOL")
  require(coponent_sate==..., "ERROR...TRANSITION_TO_ARCHIVED_STATE_INVALID")
  require(bundle.unburnedBundles() == 0, "RISKPOOL_HAS_UNBURNT_BUNDLES")
  ..
}

or a modifier archivingAllowed ... with the same effect

Reply:
I had the same thought when looking at the code. But it seems i forgot to fix it ;-)

Reply:
i will make a separate pr for this fix.